controllers/usernamespace/controller.go

```diff
@@ -448,6 +455,50 @@ func (r *CheUserNamespaceReconciler) reconcileProxySettings(ctx context.Context,
 	return err
 }
 
+func (r *CheUserNamespaceReconciler) reconcileIdleSettings(ctx context.Context, targetNs string, checluster *chev2.CheCluster, deployContext *chetypes.DeployContext) error {
+
+	noIdle := pointer.Int32Ptr(-1)
+	if checluster.Spec.DevEnvironments.SecondsOfInactivityBeforeIdling == noIdle && checluster.Spec.DevEnvironments.SecondsOfRunBeforeIdling == noIdle {
+		return nil
+	}
+	configMapName := prefixedName("idle-settings")
+	cfg := &corev1.ConfigMap{}
+
+	requiredLabels := defaults.AddStandardLabelsForComponent(checluster, userSettingsComponentLabelValue, map[string]string{
+		dwconstants.DevWorkspaceMountLabel:          "true",
+		dwconstants.DevWorkspaceWatchConfigMapLabel: "true",
+	})
+	requiredAnnos := map[string]string{
+		dwconstants.DevWorkspaceMountAsAnnotation: "env",
+	}
+
+	data := map[string]string{}
+
+	if checluster.Spec.DevEnvironments.SecondsOfInactivityBeforeIdling != noIdle {
+		data["SECONDS_OF_DW_INACTIVITY_BEFORE_IDLING"] = strconv.Itoa(int(*checluster.Spec.DevEnvironments.SecondsOfInactivityBeforeIdling))
```

Comment:
I think it is better to use `strconv.FormatInt(int64(<VALUE>), 10)` not to lost precision